HRTool/src/main/java/app/controllers/EmployeeRestController.java

```diff
@@ -1,22 +1,213 @@
 
 package app.controllers;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.ServletContext;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.rest.core.annotation.RepositoryEventHandler;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
+import app.generator.CVGenerator;
 import app.model.Employee;
+import app.model.Project;
+import app.model.ProjectInfo;
+import app.model.TagCloud;
+import app.model.TagCloudEnum;
 import app.repository.EmployeeRepository;
+import app.repository.TagCloudRepository;
+
+import com.lowagie.text.DocumentException;
 
 
 @RestController
-@RequestMapping("/employees")
+@RequestMapping("/resources/employees")

```

Comment:
You have missed the point. Employees ARE resources (contrary to the popular belief :)).
This should remain "/employees"